frontend/packages/core-api-client/src/validation/Recipient.ts

```diff
@@ -0,0 +1,30 @@
+import { FieldKind, FormWithRecord, Record } from '../types';
+import { getFieldKind } from '../utils';
+
+export const validateRecipientHierarch = (
+  formWithRecordList: FormWithRecord<Record>[],
+): void => {
+  formWithRecordList.forEach(({ form, record }, idx, arr) => {
+    if (form.id !== record.formId) throw new Error();
```

Comment:
shouldn't we pass some kind of message with the errors?

Reply:
Yeah, I think I need to add something for displaying api errors too

Reply:
so what happened to passing error messages here?